src/hooks/useLaunchableMarkets.ts

```diff
@@ -19,6 +19,16 @@ import { useDydxClient } from './useDydxClient';
 
 const ASSETS_TO_REMOVE = ['USDC', 'USDT'];
 
+export const useMarketMapPriceData = () => {
```

Comment:
we already do this in useMetadataService. Why do we need it again here?

Reply:
Ahh you're right. lemme change it up. for some reason I thought we also did filtering in the useMetadataService hook.